test/git_town_output_parser.go

```diff
@@ -0,0 +1,50 @@
+package test
+
+import (
+	"strings"
+)
+
+// GitCommandsInGitTownOutput provides the Git commands mentioned in the given Git Town output.
+func GitCommandsInGitTownOutput(output string) []string {
```

Comment:
Why are we not also returning the branch for each command? I expect we'll need it for future tests.

Reply:
I didn't want to build infrastructure unless I actually need it to keep this PR manageable. I'll add it later when it's needed. I want to understand the data format needed here better before passing such complex data around in the code base.